### PR TITLE
Seperate render methods in heading, linting issues, add Reactstrap

### PR DESCRIPTION
**Pull request by robbieaverill**

### CODE REVIEW

File: client/src/components/HistoryViewer/HistoryViewerHeading.js [hunk 1]
```diff
@@ -45,26 +45,26 @@ class HistoryViewerHeading extends Component {
           <Dropdown
             isOpen={dropdownOpen}
             toggle={this.toggle}
-            className="compare-dropdown"
-            right
+            className="compare-dropdown dropdown-menu-right"
```
Comment (raissanorth) [inline client/src/components/HistoryViewer/HistoryViewerHeading.js]: This is not required any more -> `dropdown-menu-right`

File: client/src/components/HistoryViewer/HistoryViewerVersion.js [hunk 2]
```diff
@@ -56,42 +56,74 @@ class HistoryViewerVersion extends Component {
...
   }
 
   /**
-   * Return a "clear" button to close the version, for example when used in a "detail view"
-   * context. This is shown when this version is "active", displayed with a blue background.
+   * Renders a "compare mode" button which will allow the user to start selecting versions to
+   * compare differences between. This is usually rendered in the "more actions" menu.
    *
-   * @returns {FormAction|null}
+   * @returns {FormActionComponent|null}
```
Comment (raissanorth) [inline client/src/components/HistoryViewer/HistoryViewerVersion.js]: This was actually correct before.

File: package.json [hunk 3]
```diff
@@ -74,6 +74,7 @@
     "react-redux": "^4.4.5",
     "react-router": "^2.5.2",
     "react-router-redux": "^4.0.5",
+    "reactstrap": "^5.0.0-beta",
```
Comment (raissanorth) [inline package.json]: Why did you choose a beta over a stable version?

Comment (robbieaverill) [reply package.json]: This matches what we use in silverstripe/admin =)

**Pull request merged**